vignettes/getting-started.Rmd

```diff
@@ -87,6 +89,15 @@ gimap_dataset <-  gimap::setup_data(counts = example_counts,
 
 You'll notice that this set up gives us a list of formatted data. This contains the original counts we gave `setup_data()` function but also normalized counts, and the total counts per sample. 
 
+- Raw counts: Original data
+- Counts per Sample: Add up all the counts for each sample
+- Transformed data: Contains normalized counts, counts per million (cpm)
+- Log2 CPM: log-2 transformed CPM
+- pg_metadata: paired guide metadata
```

Comment:
```suggestion
- pg_metadata: paired guide metadata - information that describes the paired-guided RNA designs. This may include the sequences used in the CRISPR design as well as what genes are targeted. 
```